Lombiq.HelpfulLibraries.AspNetCore/Security/ApplicationBuilderExtensions.cs

```diff
@@ -0,0 +1,72 @@
+﻿using Lombiq.HelpfulLibraries.AspNetCore.Security;
+using Microsoft.Extensions.DependencyInjection;
+using System;
+using System.Collections.Generic;
+
+using static Lombiq.HelpfulLibraries.AspNetCore.Security.ContentSecurityPolicyDirectives;
+using static Lombiq.HelpfulLibraries.AspNetCore.Security.ContentSecurityPolicyDirectives.CommonValues;
+
+namespace Microsoft.AspNetCore.Builder;
+
+public static class ApplicationBuilderExtensions
+{
+    /// <summary>
+    /// Adds a middleware that supplies <c>Content-Security-Policy</c> header. It may be further expanded by registering
+    /// services that implement <see cref="IContentSecurityPolicyProvider"/>.
```

Comment:
```suggestion
    /// Adds a middleware that supplies the <c>Content-Security-Policy</c> header. It may be further expanded by
    /// registering services that implement <see cref="IContentSecurityPolicyProvider"/>.
```